src/testing_system/internal/adapter/assistants/API.py

```
@@ -0,0 +1,84 @@
+                 temperature:Optional[float] = 0.9
+                 ):
+        try:
+            env_path = os.getenv("ENV_FILE_PATH", ".env")
+            load_dotenv(dotenv_path=env_path)
+            assert os.getenv("OPENAI_API_KEY") is not None, "Need an API KEY to proceed with this adapter"
+            self.model = ChatOpenAI(
+                base_url=base_url,
+                model=model,
+                temperature=temperature
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)